templates/download/risc-v/tab-6.html

```diff
@@ -1,25 +1,13 @@
 <div tabindex="0" role="tabpanel" id="qemu-tab" aria-labelledby="qemu" aria-hidden="true">
-	<div class="row u-vertically-center u-no-padding">
+	<div class="row">
 		<div class="col-6">
-			<h2>QEMU emulator</h2>
-			<h3 class="p-heading--4">Ubuntu Server preinstalled image</h3>
-			<p>
+			<h3 class="p-heading--2 u-sv3">Ubuntu Server <br class="u-hide--small">preinstalled image</h3>
+			<p class="u-sv3">
 				<a class="p-button--positive" href="https://cdimage.ubuntu.com/releases/23.04/release/ubuntu-23.04-live-server-riscv64.img.gz">Download 23.04</a>
 				<a class="p-button" href="https://cdimage.ubuntu.com/releases/22.04.2/release/ubuntu-22.04.2-live-server-riscv64.img.gz">Download 22.04.2</a>
```

Comment:
Same for this, it should be the preinstalled image link not the live-server link? https://cdimage.ubuntu.com/releases/22.04.2/release/ubuntu-22.04.2-preinstalled-server-riscv64+unmatched.img.xz